CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ else()
     cmake_minimum_required(VERSION 3.16)
 endif()
 
+add_compile_options("$<$<COMPILE_LANGUAGE:CXX>:-std=c++20>")
```

Comment:
Why is this needed?

Reply:
See the commit log.  Recent fedora updates started defaulting g++ to use --std=c++17, which triggers errors on the build.  I added this to work around them and meant to drop it.